raiden/storage/wal.py

```diff
@@ -0,0 +1,53 @@
+# -*- coding: utf-8 -*-
+from collections import namedtuple
+
+from raiden.transfer.architecture import StateManager
+
+InternalEvent = namedtuple(
+    'InternalEvent',
+    ('identifier', 'state_change_id', 'block_number', 'event_object'),
+)
+
+
+class WriteAheadLog:
```

Comment:
These will eventually replace the old `log_and_dispatch`in the `StateMachineEventHandler` and `StateChangeLog` in `transfer/log.py` I suppose?

Reply:
yes